src/table/TableBodyWrapper.jsx

```diff
@@ -69,6 +70,8 @@ const TableBodyWrapper = ({ tableData, constraints, selectionsAPI, layout, theme
                   styling={styling}
                   selState={selState}
                   selDispatch={selDispatch}
+                  tabIndex={-1}
+                  onKeyDown={(evt) => handleKeyPress(evt, rootElement, rowIndex + 1, columnIndex)}
```

Comment:
I just looked at the old table and realised that you can always navigate around it the table, at least when you attain focus by clicking a cell. So always having it here is what we should do. If we, down the line, decide to be able to disable the navigation, that's future problems :)

Reply:
No need to do that now. The whole focus in and out of the table depends on how we solve it in nebula. This is fine for now :)